bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -2300,6 +2304,8 @@ void ReplicaImp::onNewView(const std::vector<PrePrepareMsg *> &prePreparesForNew
   SeqNum firstPPSeq = 0;
   SeqNum lastPPSeq = 0;
 
+  consensus_times_.clear();
```

Comment:
I did not get at which point this function should be called.

Reply:
Since this stat is only useful on the primary, and we don't want to leak the memory for slots that don't commit on the primary, we clear it after a view change.